webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -516,7 +541,32 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
                 useChannelMentions={rest.useChannelMentions}
             />
             <div className='post-body__actions'>
-                {emojiPicker}
+                <TexteditorActions
+                    placement='middle'
+                    isScrollbarRendered={renderScrollbar}
+                >
+                    <OverlayTrigger
+                        placement='top'
+                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                        trigger={Constants.OVERLAY_DEFAULT_TRIGGER}
+                        overlay={tooltip}
+                    >
+                        <button
+                            aria-label={formatMessage({id: 'editPreview', defaultMessage: 'Preview message'})}
+                            id='EditPreviewInputTextButton'
+                            ref={previewButtonRef}
+                            onClick={togglePreviewClick}
+                            className='style--none post-action'
+                            style={{right: '24px'}}
```

Comment:
can this be done in a classname instead of style object?
className='previewButton'